templates/projects/task_presenter_editor.html

```diff
@@ -385,7 +385,7 @@ <h4 class="modal-title"><strong>{{project.name}}</strong>: {{ _('Task Presenter
      "</div>" +
      "</div>" +
      "<div id='PYBOSSA_USER_LOCALE' hidden>en</div>" +
-     "<script type=\"text/javascript\">pybossa.saveTask=function(id,answer){alert('Task '+id+' saved with answer data: '+answer+'. This is a sandbox environment and the answers will not be saved to the database!');var def=$.Deferred();def.resolve();return def;};</sc" + "ript>" +
+     "<script type=\"text/javascript\">pybossa.saveTask=function(id,answer){alert('Task '+(typeof id === 'object' ? id.id : id)+' saved with answer data: '+JSON.stringify(answer)+'. This is a sandbox environment and the answers will not be saved to the database!');var def=$.Deferred();def.resolve({});return def;};</sc" + "ript>" +
```

Comment:
Looks ok. Do you know if there is a reason why some of the lines in this file have a string break in the closing script tag? Thanks.

```js
"</sc" + "ript>"
```

Reply:
I do not know.